src/Native/Runtime/windows/CoffNativeCodeManager.cpp

```diff
@@ -296,10 +308,16 @@ void CoffNativeCodeManager::EnumGcRefs(MethodInfo *    pMethodInfo,
         codeOffset - 1 // TODO: Is this adjustment correct?
         );
 
+    ICodeManagerFlags flags = (ICodeManagerFlags)0;
+    if (pNativeMethodInfo->executionAborted)
+        flags = ICodeManagerFlags::ExecutionAborted;
+    if (IsFilter(pMethodInfo))
```

Comment:
Could you please make a matching fix in `src\Native\Runtime\unix\UnixNativeCodeManager.cpp` as well?